src/OpenRoad.cc

```diff
@@ -179,7 +179,6 @@ OpenRoad::~OpenRoad()
 void
 deleteAllMemory()
 {
-  delete OpenRoad::openRoad();
```

Comment:
Why remove this?

Reply:
It returns a pointer to a static object; which is not something that should be deleted afaik

Reply:
Agreed